src/modules/Accordion/Accordion.js

```diff
@@ -83,7 +83,7 @@ export default class Accordion extends Component {
     // Otherwise, on first render we're allowed to set state for a prop that might have a default.
     // The default prop should always win on first render.
     // This default check should then be removed.
-    if (!this.props.defaultActiveIndex) {
+    if (typeof this.props.defaultActiveIndex === 'undefined' || this.props.defaultActiveIndex === -1) {
```

Comment:
I believe this should only have the `undefined` check, true?  We only want to set the activeIndex if it was not already set.

If the prop were already set to `-1` then the auto controlled state will already be `-1` and we don't want to try to set it again.

LMK your thoughts on this.